modules/ROOT/pages/_partials/release_notes/sync-gateway-3-0-4-release-note.adoc

```diff
@@ -0,0 +1,34 @@
+== 3.0.4 -- October 2022
+
+
+=== Enhancements
+
+
+* https://issues.couchbase.com/browse/CBG-2468[++CBG-2468 [3.0.4 Backport] Maintain long-lived bucket connections for persistent config++]
+
+
+=== Issues and Resolutions
+
+==== Fixed Issues
+
+
+* https://issues.couchbase.com/browse/CBG-2448[++CBG-2448 [3.0.4 Backport] Background manager cluster aware terminator race condition++]
```

Comment:
didn't know Asciidoc macros allowed nested square braces!

Reply:
They do if you escape them.

Reply:
ah! So the `++` is just an escaping bracketing construct?